pkg/api/pluginproxy/ds_proxy.go

```diff
@@ -138,6 +138,15 @@ func (proxy *DataSourceProxy) getDirector() func(req *http.Request) {
 			req.Header.Add("Authorization", util.GetBasicAuthHeader(proxy.ds.BasicAuthUser, proxy.ds.BasicAuthPassword))
 		}
 
+		if proxy.ds.secureJsonData.custumHeaders {
+			headers := strings.Split(proxy.ds.secureJsonData.custumHeaders, ",")
+			for _,header := range headers {
+				headerData := strings.Split(header, ":")
+				req.Header.Add(headerData[0], headerData[1]) // 0 = name, 1 = value
+				logger.Info("using custom header ", headerData[0], headerData[1])
```

Comment:
This should be debug level and don't log the header value. Token's are as secret as passwords.

Reply:
done